README.md

```diff
@@ -922,6 +931,62 @@ interface INPAttribution {
 }
 ```
 
+#### `LongAnimationFrameSummary`
+
+```ts
+/**
+ * An object containing potentially-helpful debugging information summarized
+ * from the LongAnimationFrames intersecting the INP event.
+ *
+ * NOTE: Long Animation Frames below 50 milliseconds are not reported, and
+ * so their scripts cannot be included. For Long Animation Frames that are
+ * reported, only scripts above 5 milliseconds are included.
+ */
+export interface LongAnimationFrameSummary {
+  /**
+   * The number of Long Animation Frame scripts that intersect the INP event.
+   * NOTE: This may be be less than the total count of scripts in the Long
+   * Animation Frames as some scripts may occur before the interaction.
+   */
+  numScripts?: number;
+  /**
+   * The slowest Long Animation Frame script that intersects the INP event.
+   */
+  slowestScript?: PerformanceScriptTiming;
+  /**
+   * The INP phase where the longest script ran.
+   */
+  slowestScriptPhase?:
+    | 'inputDelay'
+    | 'processingDuration'
+    | 'presentationDelay';
+  /**
+   * The total blocking durations in each phase by invoker for scripts that
+   * intersect the INP event.
+   */
+  totalDurationsPerPhase?: Record<
+    'inputDelay' | 'processingDuration' | 'presentationDelay',
+    Record<string, number>
```

Comment:
I assume this was kept as a string to leave the option open to group by things other than invokers, otherwise we could use the more specific type
```suggestion
    Record<ScriptInvokerType, number>
```

Reply:
Oh I'd forgotten they'd been explicitly defined in the spec. Updated.